ConsoleApp/Classes/Stack.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+
+namespace ConsoleApp.classes
+{
+    public class Stack<T>
+    {
+        public int Count { get => ToList().Count; }
```

Comment:
that's a bad way to get `Count`. each time you need to convert stack to list just to get size